Godeps/_workspace/src/k8s.io/kubernetes/pkg/storage/cacher.go

```diff
@@ -244,6 +244,11 @@ func (c *Cacher) Delete(ctx context.Context, key string, out runtime.Object) err
 }
 
 // Implements storage.Interface.
+func (c *Cacher) DeleteDir(ctx context.Context, directory string) error {
```

Comment:
@derekwaynecarr `key` seems like a more generic parameter name than `directory` - I'm curious why you prefer `key` in this case?